db-commons/src/main/resources/reference.conf

```diff
@@ -11,8 +11,11 @@ bitcoin-s {
       //https://scala-slick.org/doc/3.3.1/api/index.html#slick.jdbc.JdbcBackend$DatabaseFactoryDef@forConfig(String,Config,Driver,ClassLoader):Database
       path = ${bitcoin-s.datadir}/${bitcoin-s.network}/
       driver = org.sqlite.JDBC
+      driverName = sqlite
       user = ""
       password = ""
+      host = localhost
+      port = 5432
```

Comment:
I don't think sqlite has the concept of a host/port as it is just a file on your filesystem?

Reply:
No, I think we should maybe rename this to something like `dbDefault` becuase it has stuff for postgres as well (`host`, `port, `user`, `pass`)